mods/gennext/README.md

```diff
@@ -236,6 +237,14 @@ New mechanic: Intrinsics:
 
   - Mismagius: Cursed Body
 
+  - Rotom: Motor Drive plus a hidden ability based on forme:
+    - Normal: Adaptability
+    - Heat: Flash Fire
+    - Wash: Storm Drain
+    - Mow: Sap Sipper
+    - Fan: Gale Wings
+    - Frost: Snow Warning
```

Comment:
No hidden abilities for now pls

Motor Drive is also too strong for Rotom-W in high-OU, isn't it?